src/main/java/lotto/domain/game/LottoNumberSet.java

```diff
@@ -21,8 +21,8 @@ public static LottoNumberSet of(List<LottoNumber> lottoNumbers) {
         return new LottoNumberSet(new TreeSet<>(lottoNumbers));
```

Comment:
```suggestion
        return new LottoNumberSet(Collections.unmodifiableSet(new TreeSet<>(lottoNumbers)));
```
지난 피드백에서 이 부분이 빠져있는 것 같네요 ㅎㅎ value() 메서드에서 매번 unmodifable로 만드는것 보다 처음부터 unmodifable인 상태로 있는 게 더 자연스러워 보입니다 :)
changeLog에서 private 생성자가 안보여서 여기에 unmodifable 로직을 달았는데, 사실 생성자에서 해주면 되겠죠 :)